packages/p2p/src/components/app.jsx

```diff
@@ -50,33 +69,43 @@ class App extends Component {
         }
 
         return (
-            <Fragment>
-                {/*
-                    App can overwrite the styles by passing css variables to className deriv-api
-                    you can refer to deriv-shared/themes for the css variables that are used in deriv-app as well as p2p
-                */}
-                <main className={ classNames('deriv-p2p', className) }>
-                    <Tabs active_index={active_index} top>
-                        {/* TODO [p2p-uncomment] uncomment this when sell is ready */}
-                        {/* <div label={localize('Buy / Sell')}> */}
-                        <div label={localize('Buy')}>
-                            <BuySell navigate={this.redirectTo} params={parameters} />
-                        </div>
-                        {/* TODO: [p2p-replace-with-api] Add 'count' prop to this div for notification counter */}
-                        <div label={localize('Orders')}>
-                            <Orders navigate={this.redirectTo} params={parameters} />
-                        </div>
-                        {/* TODO [p2p-uncomment] uncomment this when my ads is ready */}
-                        {/* <div label={localize('My ads')}> */}
-                        {/*    <MyAds navigate={this.redirectTo} params={parameters} /> */}
-                        {/* </div> */}
-                        {/* TODO [p2p-uncomment] uncomment this when profile is ready */}
-                        {/* <div label={localize('My profile')}>
-                            <MyProfile navigate={this.redirectTo} params={parameters} />
-                        </div> */}
-                    </Tabs>
+            <AgentProvider value={this.state.is_agent}>
```

Comment:
keep it as an object to make it easier to extend in the future? (if we ever want to use the AgentContext for other properties then is_agent)

obs if this is changed the usage will be:
` const { is_agent } = React.useContext(AgentContext);`

```suggestion
            <AgentProvider value={{ is_agent: this.state.is_agent }}>
```